Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2078,33 +2079,36 @@ public Expression resolveIdentifier(String identifier, boolean mustResolve) {
         // 10: The name of a property on a specific context
         // 11: An unresolved identifier error is thrown
 
+        //Collect every match, issue warning if multiple matches were made, but return initial match:
+        List<Expression> matchCollection = new ArrayList<>();
+
         // In a type specifier context, return the identifier as a Literal for resolution as a type by the caller
         if (inTypeSpecifierContext()) {
-            return this.createLiteral(identifier);
+            matchCollection.add(this.createLiteral(identifier));
```

Comment:
Here the description would be `String.format("%s resolved as a potential type name")`